### DISCUSSION TITLE
Containerising this app

### DISCUSSION THREAD

**Discussion by zakisahal** [ivanlori/Vuemmerce#42] [General]
Hey, 

I'm practicing deploying container images on to the cloud via aws ecs service, I would like to containerise this app using docker but im struggling. Any tips?

**Comment by ScribblerCoder** [comment]
hey I just came upon this repo. Use this run a dev version of Vuemmerce with hot autoreloading

```
FROM node:16.20.2-bullseye-slim as build

# COPY frontend
COPY Vuemmerce-v.1.1.0 /frontend

WORKDIR /frontend

# install dependencies
RUN yarn install

CMD ["/usr/local/bin/yarn", "dev"]
```
